sdk-commons/src/main/java/org/openmrs/maven/plugins/model/BaseSdkProperties.java

```diff
@@ -150,6 +150,18 @@ public List<Artifact> getConfigArtifacts() {
 		return artifactList;
 	}
 
+	public List<Artifact> getContentArtifacts() {
+        List<Artifact> artifacts = new ArrayList<>();
+        for (Object keyObject: getAllKeys()) {
+            String key = keyObject.toString();
+            if (key.startsWith(TYPE_CONTENT + ".")) {
+                String artifactId = key.substring(TYPE_CONTENT.length() + 1);
+                artifacts.add(new Artifact(artifactId, getParam(key), Artifact.GROUP_CONTENT, Artifact.TYPE_ZIP));
```

Comment:
The pattern here should be the same as with `getModuleArtifacts()`, i.e., using `checkIfOverwritten()` for most of the properties

Reply:
Used getOwaArtificats as its dealing with ZIP file like content packages, will update to use getModuleArtifacts() pattern